### DISCUSSION TITLE
Disable player at start page

### DISCUSSION THREAD

**Discussion by msman0** [owncast/owncast#2971] [Q&A]
Hi, is it possible to disable the player at the owncast server start page? I only want to embed the stream at a separate page.
Thank you!

**Comment by gabek** [comment]
Could you elaborate a bit? What do you mean by disable the player on the start page? It doesn't autoplay so it won't be playing just by visiting your Owncast page.

**Comment by msman0** [comment]
I mean that it is not possible to open the url from the owncast server e.g. stream.domain.com and see the stream. It should be only possible from an other domain (embedded iframe). domain2.com

**Reply by gabek** [reply]
You'd want to do that in your proxy and micromanage the paths you want to allow and deny, maybe with some origin filtering. I don't have any specific instructions for you, but none of this would be specific to Owncast. Do some searching for using your web proxy of choice to restrict and allow certain paths on your web server.
